content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md

```diff
@@ -404,6 +418,87 @@ db:
     database: <POSTGRESQL_CLUSTER_DB>
 ```
 
+#### State Manager Databases
+
+Corda requires one or more PostgreSQL database instances for the persistence of state between worker types. These are referred to as state manager databases. R3 recommends that you deploy separate isolated state manager database instances for the following workers:
```

Comment:
`persistence of state between worker types` isn't quite correct, it probably should be 

"Corda requires one or more PostgreSQL database instances for the persistence of state."